e2e-test/features/RunTimeSLT.feature

```diff
@@ -0,0 +1,162 @@
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+Feature:Run-Time SLT Scenario
+
+  @SLT @Run-Time @Sanity
+  Scenario: User is able to create MTID and update CDF_R_SLT_SETTINGS program
+    Given User crates new MTID on "Automation" SAP
+    When User updates mtid config in CDF_R_SLT_SETTINGS program
+
+  @SLT @Run-Time @Sanity
+  Scenario Outline: User configured SLT job for table containing all data types
+    Given User crates new MTID on "Automation" SAP
+    When User updates mtid config in CDF_R_SLT_SETTINGS program
+    Then Update mass transfer id table: "<table>" job mode to "stop_load"
+    Given Open CDF replication and initiate pipeline creation
+    When User fetches the record count of table : "<table>" from SAP
+    Then Delete GCS folder for mass transfer id table "<table>"
+    Then Drop target BigQuery table "<table>"
+    Then Update mass transfer id table: "<table>" job mode to "<mode>"
+    When Source is SAP SLT fill connection parameters
+    Then Click on next
+    Then Select Table "<table>"
+    Then Click on next
+    Then Enter the BigQuery Properties for slt datasource
+    Then Click on next
+    Then Click on next
+    Then Click on next
```

Comment:
We have three steps clicking on the Next button. I believe, they are clicking the Next button element on three different pages. It will make test more readable if the step can be enhanced to take the page name and print it in the logs. 

Then Click on next button on "String"